app/models/merchant.rb

```diff
@@ -40,7 +40,22 @@ def filter_item_status(status_enum)
     items.where(status: status_enum)
   end
 
+  def self.top_five_merchants
+    joins(invoices: :invoice_items)
+      .joins(:transactions)
```

Comment:
Nice query as a class method.  I didn't know you could chain joins like that.  I wonder if its possible to do that with a different syntax.

Reply:
@kbhoffmann Yes I meant to ask for feedback on that!  I tried a few different things such as `joins(invoices: :transactions, invoices :invoice_items)` but couldnt get the same result

Reply:
Interesting.  I guess as long as it works!  We can always try refactoring at a later time.  I think Sierra or Hannah, posted a question about syntax in codehelp.  I need to look at that thread closer.